templates/default/deploy-scripts/src/main.rs

```diff
@@ -0,0 +1,170 @@
+pub mod deployer;
+use anyhow::Context;
+use clap::Parser;
+use concordium_rust_sdk::{
+    common::types::Amount,
+    endpoints::{self, RPCError},
+    smart_contracts::{
+        common::{self as contracts_common, NewContractNameError, NewReceiveNameError},
+        types::{OwnedContractName, OwnedParameter, OwnedReceiveName},
+    },
+    types::{
+        smart_contracts::{ExceedsParameterSize, WasmModule},
+        transactions,
+        transactions::send::GivenEnergy,
+    },
+    v2,
+};
+use itertools::Itertools;
+
+use concordium_rust_sdk::types::transactions::InitContractPayload;
+use deployer::{Deployer, ModuleDeployed};
+use hex::FromHexError;
+use std::{
+    io::Cursor,
+    path::{Path, PathBuf},
+};
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum DeployError {
```

Comment:
Since this is an application, is it important to have all of these refined errors?

Would it not be enough to just use `anyhow::Error` everywhere?

Reply:
That would make it simpler to write and simpler to look at as well.